doc/alertmanager-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,310 @@
+              rate(alertmanager_notification_latency_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
+          ||| % $._config, legendFormat='{{integration}} - Median'
+        ))
+        .addTarget(prometheus.target(
+          |||
+            rate(alertmanager_notification_latency_seconds_sum{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            /
+            rate(alertmanager_notification_latency_seconds_count{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+          ||| % $._config, legendFormat='{{integration}} - Average'
+        ));
+
+
+      local silences =
+        graphPanel.new(
+          'Silences',
+          datasource='$datasource',
+          span=4,
+          format='none',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('sum(alertmanager_silences{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}) by (state)' % $._config, legendFormat='{{ state }}'));
+
+      local silenceQueries =
+        graphPanel.new(
+          'Queries',
+          datasource='$datasource',
+          span=4,
+          format='reqps',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('rate(alertmanager_silences_queries_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])' % $._config, legendFormat='Queries'))
+        .addTarget(prometheus.target('rate(alertmanager_silences_query_errors_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])' % $._config, legendFormat='Errors'));
+
+
+      local silenceQueryDuration =
+        graphPanel.new(
+          'Queries Duration',
+          datasource='$datasource',
+          span=4,
+          format='s',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target(
+          |||
+            histogram_quantile(0.99,
+              rate(alertmanager_silences_query_duration_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
+          ||| % $._config, legendFormat='99th Percentile'
+        ))
+        .addTarget(prometheus.target(
+          |||
+            histogram_quantile(0.50,
+              rate(alertmanager_silences_query_duration_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
+          ||| % $._config, legendFormat='Median'
+        ))
+        .addTarget(prometheus.target(
+          |||
+            rate(alertmanager_silences_query_duration_seconds_sum{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            /
+            rate(alertmanager_silences_query_duration_seconds_count{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+          ||| % $._config, legendFormat='Average'
+        ));
+
+      local silencesGossips =
+        graphPanel.new(
+          'Gossips',
```

Comment:
This is only about gossiping silences, which is fairly rare (only when a silence is created or updated by a user). `alertmanager_nflog_gossip_messages_propagated_total` is probably more interesting, but perhaps still too low-level to put it on the dashboard.

How about first refining the panels above and leave out gossiping for now?